rust/main/chains/hyperlane-radix/src/mailbox.rs

```diff
@@ -259,13 +273,67 @@ impl Mailbox for RadixMailbox {
     }
 }
 
-/// Data required to check if a message was delivered on-chain for Radix chain
-#[derive(Clone, Debug, Deserialize, Serialize)]
-pub struct DeliveredCalldata {
+/// Data required to send a tx on radix
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RadixTxCalldata {
     /// Address of mailbox (already encoded)
     pub component_address: String,
     /// Method to call on mailbox
     pub method_name: String,
     /// parameters required to call method
     pub encoded_arguments: Vec<u8>,
 }
```

Comment:
We don't want to create dependency on Mailbox in Lander, so, we cannot have this struct in `mailbox.rs`. We avoided this problem in EVM and SVM by serialising either Sealevel `Instruction` directly or a tuple of EVM-spetific structs like `TypedTransaction` and `Function`. We can either define a tuple here as well or we need to move this struct somewhere. Lander depends on RadixProvider, so, we can keep `RadixTxCalldata` there, I think.